packages/config/config/devices/0x027a/zen24_3.0-3.255.json

```
@@ -240,7 +236,7 @@
 					"value": 1
 				},
 				{
-					"label": "DT Disabled, Single Tap to Full / Max Brightness",
+					"label": "DT Disabled, Single Tap to Full",
```

```suggestion
					"label": "Double Tap Disabled, Single Tap to Full Brightness",
```
Don't think that works if abbreviated too much. Please also edit the option for value 1.